remote_attestation/java/src/com/google/oak/remote_attestation/ClientHandshaker.java

```diff
@@ -188,36 +187,50 @@ public AeadEncryptor getEncryptor() throws RuntimeException {
    *   {@code expectedTeeMeasurement}.
    */
   private Boolean verifyAttestationInfo(Message.ServerIdentity serverIdentity) throws IOException {
-    byte[] additionalInfo = serverIdentity.getAdditionalInfo();
-    // Ensure additional info was supplied.
-    if (additionalInfo == null || additionalInfo.length == 0) {
-      logger.log(Level.WARNING, "Configuration data cannot be empty");
+    byte[] additionalAttestationData = serverIdentity.getAdditionalAttestationData();
+
+    // Ensure additional attestation data was supplied.
+    if (!verifyAdditionalAttestationData(additionalAttestationData)) {
+      logger.log(Level.WARNING, "Additional attestation data was invalid");
       return false;
     }
 
     // Check the hash of the public key and additional info
-    AttestationInfo attestationInfo = AttestationInfo.parseFrom(
```

Comment:
I'm not sure why the ephemeral public key was previously not included. @ipetr0v do you remember if that was intended?

Reply:
Ok thanks, I think we should probably re-think this part anyways (adding or removing things that get signed). The good news is that now this is defined in a single place (well, two, one for Rust and one for Java).